### DISCUSSION TITLE
Build dependencies

### DISCUSSION THREAD

**Discussion by szepeviktor** [npm/feedback#940] [CLI]
There is a need for a 3rd type of dependency.
When build tools live in devDeps they cannot be used in production.
When build tools live in deps they stay in production but in vain: no build tools are necessary when the application is running.

Have you ever needed buildDeps?

**Comment by szepeviktor** [comment]
An article about work-arounds https://withblue.ink/2020/06/07/is-this-a-dependency-or-devdependency.html

**Comment by ljharb** [comment]
Builds shouldn't ever be done in production; they're dev deps.

If it needs to run in production, it's *not* a dev dep, it's a prod dep.
